Tune flags to build libv8 on armel
 -mno-thumb-interwork to allow compilation on default debian armel target (-march=armv4t), see
  http://code.google.com/p/v8/issues/detail?id=590
 -fno-tree-sink to workaround a gcc armel bug, see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=39604
 -Wno-psabi to fix gcc warnings on armel, see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=42748

--- a/SConstruct
+++ b/SConstruct
@@ -202,7 +202,8 @@
       },
       'unalignedaccesses:off' : {
         'CPPDEFINES' : ['CAN_USE_UNALIGNED_ACCESSES=0']
-      }
+      },
+      'CCFLAGS':      ['-mno-thumb-interwork', '-fno-tree-sink', '-Wno-psabi']
     },
     'simulator:arm': {
       'CCFLAGS':      ['-m32'],
